### ISSUE TITLE
Command line prompt text is empty

### ISSUE THREAD

**Issue by oktupol** [vimperator/vimperator-labs#754]
##### Issue type:

- Bug report

##### Version:

```
Vimperator: 3.16.0 (created: 2017/02/01 00:55:15)
Firefox: Mozilla/5.0 (Windows NT 10.0; WOW64; rv:53.0) Gecko/20100101 Firefox/53.0
```

##### Description:

Command line prompt text (that purple label next to the input when in find mode, not sure if this is the correct name) is empty.  
For example, in hint mode, instead of a purple label saying "Follow hint" left of the command prompt, there is just an empty purple label.

##### Expected behavior:

A text inside the purple label when in find mode or in hint mode.

##### Steps to reproduce:

Press `/` or `f`

**Comment by SammysHP** [comment]
I discovered this independently and fixed it in 5f329c463b4f9268f8705b1280348881eb2c7fa3.

**Comment by SammysHP** [comment]
See also https://bugzilla.mozilla.org/show_bug.cgi?id=1331296.
